INSTALL.md

```diff
@@ -5,7 +5,7 @@
 To run the commands in the README.md the following items are required
 
 1. Apache httpd server 2.4
-1. Python libraries `pip install flask pybind11 celery connexion`
+1. Python libraries `pip install flask pybind11 celery redis connexion[swagger-ui]`
```

Comment:
why do we need redis?

Reply:
redis is used for the Celery task queue. Celery does not ship with it automatically

Reply:
Would it be better to add the `pip install ...` to the chapter where it is imported?

Reply:
Yes, please